external/test-runtime/optional.json

```diff
@@ -0,0 +1,26 @@
+{
+  "frameworks": {
+    "net45": {
+      "dependencies": {
+        "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
+        "Microsoft.DotNet.UAP.TestTools": "1.0.2",
+        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25131-00"
+      }
+    }
+  },
+  "runtimes": {
```

Comment:
And the `optional.json` in buildtools is just copied from the tool-runtime/project.json. :smile: None of the packages currently have any native assets.